resources/views/Pages/find.blade.php

```diff
@@ -9,33 +9,31 @@
             </div>
 
             <div class=" row text-center mt-5">
-                <h3><button class='btn btn-dark' >
-                    <a class="" onclick="locatorButtonPressed()" style="color: white">
+                <h3>
```

Comment:
This file should not be change in this merge since the file in this commit is older version than this file in the master